lib/tapioca/commands/abstract_gem.rb

```diff
@@ -307,6 +433,74 @@ def merge_with_exported_rbi(gem, file)
         say_error("\n\n  RBIs exported by `#{gem.name}` contain errors and can't be used:", :yellow)
         say_error("Cause: #{e.message} (#{e.location})")
       end
+
+      sig { params(annotation: RBI::Tree, file: RBI::File).void }
+      def merge_with_annotation(annotation, file)
+        file.root = RBI::Rewriters::Merge.merge_trees(
+          file.root,
+          annotation,
+          keep: RBI::Rewriters::Merge::Keep::LEFT,
```

Comment:
I wonder if that should be a CLI option?

Reply:
I'm having difficulty coming up with how we communicate this option. Would the possibilities be annotation or rbi as the base?